modules/ROOT/pages/mongodb/mongodb-connector-6-0-upgrade-migrate.adoc

```diff
@@ -0,0 +1,326 @@
+
+| Update Documents
+| N/A
+a| JSON that contains the following structure:
+
+* Int Matched
+* Int Modified
+* JSON upsertedId
+
+| Remove Documents
+| Query: JSON Object
+| N/A
+
+| Count Documents
+| Query: JSON Object
+| N/A
+
+| Find Documents
+| Query: JSON Object
+| Resolved dynamically based on the selected value of the collection parameter. The connector uses the structure of the last document in the given collection.
+
+| Create File
+| N/A
+a| A GridFS JSON object with the following attributes:
+
+* JSON Id
+* String fileName
+* Long Length
+* Int ChunkSize
+* Datetime uploadDate
+* JSON Metadata
+
+| Find Files
+| N/A
+a|List<InputStream>
+List of JSON objects with the following attributes:
+
+* JSON Id
+* String fileName
+* Long Length
+* Int ChunkSize
+* Datetime uploadDate
+* JSON Metadata
+|===
+
+== Removed Operations
+
+The following operations were removed from the MongoDB connector:
+
+[%header,cols="15%,35%,15%,35%"]
+|===
+2+|Removed from MongoDB 5.x 2+| Can be reproduced in MongoDB 6.0.0 through
+|Name	| Description	|Name	|Description
+|Incremental dump | Executes an incremental dump of the database. | Dump | Use the Dump operation.
+|Restore | Takes the output from the dump and restores it | Restore from file or Restore from directory | Restore from file or Restore from directory take the output from the dump file or directory and restore it.
+|Update documents by function | Update documents using a Mongo function | N/A | N/A
+|Update documents by functions | Update documents using one or more Mongo functions | N/A | N/A
+|Find one and update document | Finds and updates the first document that matches a given query | N/A | This operation's functionality can be reproduced by combining other operations.
+|Save document | Inserts or updates a document based on its object ID | N/A | N/A
+|Find one document | Finds the first document that matches a given query | Find documents | Set the Limit=1 to return one document.
+|Execute generic command | Executes a generic command on the database | Execute command | Executes a command on the database
+|List files | Lists all files that match the given query and sorts them by filename | N/A | N/A
+|Find one file | Returns the first file that matches the given query | Find files | Specifying a specific file to query
+|===
+
+== Changes in Input Sources
+
+MongoDB 6.0.0 has one input source, Object Listener, which retrieves all of the created documents that belong to a specific collection.
+
+The Delete Sources and Update Sources input sources were removed.
+
+=== Metadata in Object Listener
+
+[%header,cols="50%,50%"]
+|===
+|Input Metadata	|Output Metadata
+
+| N/A
+| The connector resolves output metadata dynamically based on the selected value of the Collection Name parameter. The connector reads the last document of the given collection and uses the document's structure as output metadata.
+|===
+
+== Requirements and Limitations
+
+== Upgrade Prerequisites
```

Comment:
Are these supposed to be at the same level? Is the first one just empty? Clarify.